tests/events.py

```diff
@@ -10,10 +7,3 @@ class SomeEvent(Event):
 
 class AnotherEvent(Event):
```

Comment:
Why did you left it here and moved other stuff out?

Reply:
Cause there are still dependencies on other parts of tests. 

I will remove it after refactoring the rest.